doc/developers/advanced_installation.rst

```diff
@@ -242,12 +242,17 @@ The above commands assume that you have the Python installation folder in your
 PATH environment variable.
 
 You will need `Build Tools for Visual Studio 2017
-<https://visualstudio.microsoft.com/de/downloads/>`_.
+<https://visualstudio.microsoft.com/downloads/>`_.
+
+.. warning::
+	You DO NOT need to install Visual Studio 2019. You only need the "Build Tools for Visual Studio 2019", under "All downloads" -> "Tools for Visual Studio 2019". 
 
 For 64-bit Python, configure the build environment with::
 
     SET DISTUTILS_USE_SDK=1
-    "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64
+    "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64
```

Comment:
Note: this is fine for this line to be longer than 80 characters, but for text paragraphs it's better to respect the 80 chars limit.